sway/tree/container.c

```diff
@@ -547,6 +547,20 @@ bool container_has_child(struct sway_container *con,
 	return container_find(con, find_child_func, child);
 }
 
+int container_count_descendants_of_type(struct sway_container *con,
+		enum sway_container_type type) {
+	int children = 0;
+	if (con->type == type) {
+		children++;
+	} else if (con->children) {
```

Comment:
This works fine when passing `C_VIEW` to the function, but if we were to try to pass `C_CONTAINER` then it wouldn't descend into the container's children.